style.css

```diff
@@ -258,10 +262,92 @@ p {
   font-size: 12px;
 }
 
-.recent-works .work-cards .project-one {
-  background-image: url(./icons/Img-Plaholder.png);
-  background-repeat: no-repeat;
-  background-size: cover;
+/* project-modal */
+.desktop-view-btn {
+  display: none;
+}
+
+.mobile-view-btn {
+  display: flex;
+  gap: 12px;
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 999;
+  left: 6%;
+  top: 2%;
+  width: 80%;
+  height: 94vh;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  text-align: left;
+  flex-direction: column;
+  padding: 10px;
+  overflow: auto;
+}
+
+.modal .close-modal-btn {
+  align-self: flex-end;
+  border: none;
+  background-color: #fff;
+}
+
+.modal .project-img {
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.modal .project-img img {
+  width: 100%;
```

Comment:
Desktop popup:

- [ ] Kindly make sure to follow the height and width for the popup as per the Figma design.
- [ ] Kindly make sure that when you click on `see project `and a popup appears, the background of the page is blurred and follows the colour displayed as per the Figma design.
- [ ] Kindly make sure you have the correct spacing to the top, right, left and bottom of the popup as per the Figma design.
- [ ] Kindly make sure to follow the correct size for buttons as per the Figma design.
- [ ] Kindly reduce the spacing between the title of the popup and technologies as per the Figma design.
- [ ] Kindly follow the font size and spacing for the popup paragraph, and title.

Your design:
<img width="1663" alt="Screenshot 2022-05-11 at 10 34 56 PM" src="https://user-images.githubusercontent.com/70269507/167931651-4f8d1f82-1da1-41d8-af15-db07d31b82fe.png">


Figma design:
<img width="657" alt="Screenshot 2022-05-11 at 10 30 24 PM" src="https://user-images.githubusercontent.com/70269507/167931301-af4edeca-f961-4502-be24-d0bef6371dd0.png">